src/CommonFiles/Oryx.ruleset

```diff
@@ -21,6 +21,9 @@
     <Rule Id="SA1611" Action="None" /> <!-- ElementParametersMustBeDocumented -->
     <Rule Id="SA1615" Action="None" /> <!-- ElementReturnValueMustBeDocumented -->
     <Rule Id="SA1618" Action="None" /> <!-- GenericTypeParametersMustBeDocumented -->
+    
+    <!-- Documentation text should end in a period -->
+    <Rule Id="SA1629" Action="None" />
```

Comment:
Here did we make ruleset stricter or looser? Curious does `Action="None"` mean that we start to ignore such cases?

Reply:
Yes, this change ignores the errors that appeared when documentation text didn't end in a period. I wanted to document some methods that I wrote, and this error kept appearing. I disabled it since its my personal preference to allow docs without punctuation, but if people have strong feelings I will happily revert this change and add punctuation to the method docs